### ISSUE TITLE
@qp-message branch- ForBES message

### ISSUE THREAD

**Issue by mkrishan** [kul-forbes/ForBES#6]
ForBES gives messgae-"reached optimum" with adaptive nama and panoc on maros-meszaros- but problem not solved- for example choose two sets DTOC3 & DUALC2 with p_test_maros_meszaros.m

**Comment by lostella** [comment]
Are those problems strongly convex? If that is not the case, unfortunately
NAMA will have a hard time at solving them.

**Comment by mkrishan** [comment]
Yes, they are made strongly convex using regularization to Hessian. Could this message be maintained if the problem is not actually solved in ForBES? Actually the same type of data-sets fails line-search in a very naive implementation of NAMA.
